### PR TITLE
Fix Docker and Kubernetes import issues

### LINKED ISSUE #861: Error cannot resolve module 'ballerinax/docker'
### Description:
Error while docker build

### Steps to reproduce:
Enable docker configurations [docker.dockerConfig] in deployment-config.toml

Errors:
`error: wso2/petstore:0.1.0::listeners.bal:7:1: cannot resolve module 'ballerinax/docker'
error: wso2/petstore:0.1.0::listeners.bal:66:1: undefined module 'docker'
error: wso2/petstore:0.1.0::listeners.bal:66:1: undefined annotation 'Config'
error: wso2/petstore:0.1.0::listeners.bal:75:5: undefined module 'docker'
error: wso2/petstore:0.1.0::listeners.bal:75:5: undefined annotation 'CopyFiles'`

`error: wso2/petstore:0.1.0::listeners.bal:78:20: mismatched input 'source'. expecting '}'
error: wso2/petstore:0.1.0::listeners.bal:79:27: invalid token '"/home/ballerina/conf/micro-gw.conf"'
error: wso2/petstore:0.1.0::listeners.bal:79:26: mismatched input ':'. expecting {'[', '?', '|', Identifier}
error: wso2/petstore:0.1.0::listeners.bal:80:35: mismatched input ':'. expecting '='`

### Affected Product Version:
jballerina branch
toolkit 3.0.2

### PR DESCRIPTION
**Pull request by AmaliMatharaarachchi**
### Purpose
Fix https://github.com/wso2/product-microgateway/issues/861 - Error cannot resolve module 'ballerinax/docker'

### Issues
Fixes #861

### Automation tests
 - Unit tests added: No
 - Integration tests added: No

### Tested environments
MacOS catalina

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

**Comment by praminda** [comment]
@AmaliMatharaarachchi please complete the pull request template and don't delete any of the fields. Those are required for some automated checks we are performing to validate a PR.

**Comment by AmaliMatharaarachchi** [comment]
@praminda Edited as requested.

File: components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/config/CopyFile.java [hunk 1]
```diff
@@ -23,7 +23,7 @@
  * annotation.
  */
 public class CopyFile {
-    private String source;
+    private String sourceFile;
```
Comment (praminda) [inline components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/config/CopyFile.java]: @Rajith90 @hasuniea @AmaliMatharaarachchi 
hmm,
This change depends on how we are going to version the first jballerina based mgw release. This change makes older deployement-config files invalid isn't it?
If we are good to do 4.0.0 i'm ok with this change, if not -1

Comment (AmaliMatharaarachchi) [reply components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/config/CopyFile.java]: Reverted the change in https://github.com/wso2/product-microgateway/pull/862/commits/bb187e88405824d88734578b6a95503ed078babb, https://github.com/wso2/product-microgateway/pull/862/commits/013f9019e5a9ed2f874b9ef63696e554486d1fcb

**Pull request merged**